doc/content/integrations/mqtt/_index.md

```diff
@@ -13,7 +13,12 @@ weight: 30
 
 ## Multi-Tenancy
 
-Note that on multi-tenant environments such as Cloud, application IDs and other endpoints include the tenant ID, e.g `app1@tenant1`. On single tenant environments such as Open Source, the tenant ID can be removed, i.e `app1`.
+On multi-tenant environments such as Cloud, application IDs and other endpoints include the tenant ID, e.g `app1@tenant1`. On single-tenant environments such as Open Source, the tenant ID can be removed, i.e `app1`.
```

Comment:
TTN is single tenant and it cannot be removed.

I would say that one should always pass a tenant ID, except with The Things Stack Open Source.